cypress/support/commands/extract-biblio-id.ts

```diff
@@ -11,7 +11,7 @@ export default function extractBiblioId(subject: undefined | JQuery<HTMLElement>
 
     cySubject = cy.wrap(subject, { log: false })
   } else {
-    cySubject = cy.get('.list-group-item-main', { log: false })
+    cySubject = cy.get('.c-thumbnail-and-text', { log: false })
```

Comment:
I would replace this with this selector: `.card > .card-body > .list-group > .list-group-item`. Did some quick analysis and that should work during import and in the overview for selecting the individual publications/datasets.